### ISSUE TITLE
shellcheck does not report stray break in if statements

### ISSUE THREAD

**Issue by brother** [koalaman/shellcheck#78]
brother /tmp$ cat test.sh

``` bash
#!/bin/bash

a="hello"
if [[ $a != "bye" ]]; then
    echo "say something"
    break
else
    echo "ok"
fi
```

brother /tmp$ shellcheck test.sh 
brother /tmp$ echo $?
0
brother /tmp$ ./test.sh
say something
./test.sh: line 6: break: only meaningful in a `for',`while', or `until' loop

(forgot to include my mailaddress when I reported this via shellcheck.net earlier)
